ci/prow/config.yaml

```diff
@@ -1094,7 +1094,7 @@ presubmits:
         - "--postsubmit-job-name=post-knative-pkg-go-coverage"
         - "--artifacts=$(ARTIFACTS)"
         - "--profile-name=coverage_profile.txt"
-        - "--cov-target=./apis/ ./configmap/"
+        - "--cov-target=."
```

Comment:
1) it automatically exclude vendor folder, but not test
2) let's confirm with @jessiezcc  and see if we want to apply this pattern to all repos.

Reply:
can u apply gitattribute to a folder?

Reply:
I suggest that you be consistent with specifying excluded directories in the tool to avoid confusion and conflict, and thus not automatically exclude `//vendor` and leave that (and `//test`, in this case) for `.gitattributes` just like https://github.com/knative/serving/blob/665012885601188d00e54f4da8fe539cd28765be/.gitattributes#L11

Reply:
Can we get the coverage and then prune it back :)

Reply:
@jessiezcc yes

Reply:
@steuhs yes